docs/jupiterone.md

```diff
@@ -96,21 +96,30 @@ https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources | Entity `_type` | Entity `_class` |
-| --------- | -------------- | --------------- |
-| Account   | `acme_account` | `Account`       |
-| User      | `acme_user`    | `User`          |
-| UserGroup | `acme_group`   | `UserGroup`     |
+| Resources          | Entity `_type`             | Entity `_class` |
+| ------------------ | -------------------------- | --------------- |
+| Account            | `asana_account`            | `Account`       |
+| Project            | `asana_project`            | `Project`       |
+| Project Membership | `asana_project_membership` | `AccessRole`    |
+| Team               | `asana_team`               | `Team`          |
+| User               | `asana_user`               | `User`          |
+| Workspace          | `asana_workspace`          | `Organization`  |
 
 ### Relationships
 
 The following relationships are created:
 
-| Source Entity `_type` | Relationship `_class` | Target Entity `_type` |
-| --------------------- | --------------------- | --------------------- |
-| `acme_account`        | **HAS**               | `acme_group`          |
-| `acme_account`        | **HAS**               | `acme_user`           |
-| `acme_group`          | **HAS**               | `acme_user`           |
+| Source Entity `_type`      | Relationship `_class` | Target Entity `_type`      |
+| -------------------------- | --------------------- | -------------------------- |
+| `asana_account`            | **HAS**               | `asana_workspace`          |
+| `asana_project`            | **HAS**               | `asana_project_membership` |
+| `asana_project_membership` | **HAS**               | `asana_user`               |
```

Comment:
I'd change this slightly to be:

`asana_user` -ASSIGNED-> `asana_project_membership`.

`asana_project_membership` acts as permissions here and generally users have assigned certain permissions.

Reply:
Ok, will do.

Reply:
The `/project_memberships` payload roughly looks like this.
```
{
            "gid": "{gid_number}",
            "project": {
                "gid": "{gid_number}",
                "resource_type": "project"
            },
            "user": {
                "gid": "{gid_number}",
                "resource_type": "user"
            },
            "write_access": "full_write",
            "resource_type": "project_membership"
        },
```
I'm also planning to add a `PROJECT_MEMBERSHIP ALLOWS PROJECT` in this case.